app/services/ReferenceDataDynamicRadioService.scala

```diff
@@ -37,64 +36,54 @@ class ReferenceDataDynamicRadioService @Inject() (
       .getIncidentCodes()
       .map(_.sortBy(_.code.toLowerCase))
 
-  def getTypesOfLocation(userAnswers: UserAnswers)(implicit hc: HeaderCarrier): Future[Seq[TypeOfLocation]] =
-    userAnswers.get(IsSimplifiedProcedurePage) match {
-      case Some(Normal) =>
-        referenceDataConnector
-          .getTypesOfLocation(userAnswers)
-          .map(_.sortBy(_.`type`.toLowerCase))
-      case _ =>
-        referenceDataConnector
-          .getTypesOfLocation(userAnswers)
-          .map(_.sortBy(_.`type`.toLowerCase))
-          .map(
-            x => filterLocationUserAnswers(x)
-          )
-    }
+  def getTypesOfLocation(userAnswers: UserAnswers)(implicit hc: HeaderCarrier): Future[Seq[TypeOfLocation]] = {
```

Comment:
Could we do a getMandatoryPage on IsSimplifiedProcedurePage at the controller level?